js/dashboard0.js

```diff
@@ -0,0 +1,111 @@
+// Get your own Articles
+
+window.onload = function (){
+    firebase.auth().onAuthStateChanged( user => {
+        console.log("Status Changed", user);
+    
+        if(user){
+            console.log(user.uid)
+            // let currentUser = user.uid
+            console.log(user.email)
+        }
+
+    this.getArticles(user);
+    })
+    updateForm()
+}
+
+let main = document.getElementById("dash1");
+let grid = document.getElementById("grid-container")
+    
+    function getArticles(user){
+        db.ref("Blogs/").limitToLast(10).once("value", function (snapshot){
+            
+            snapshot.forEach(
+                function (childSnapshot){
+    
+                    let key = childSnapshot.key
+                    let articles = childSnapshot.val()
+                    console.log(key);
+                    let tableBody = `  
+                        <div>${articles.post_title}</div>
+                        <div>${articles.post_body}</div>
+                        <div>${articles.published}</div>
+                        <div>
+                        <button class="delete" id="${key}" onClick="delete_post(this.id)">Delete</button>
+                        <button class="update" id="${key}" onClick="openFormModal(this.id)">Update</button></div>`;
+                        
+                    console.log(tableBody)
+                    grid.innerHTML += tableBody
+                }
+            )
+        })
+    }
+
+const delete_post = (key) => {
+    // console.log(key)
+    db.ref("Blogs/" + key).remove();
+    // db.ref("comments/" + key).remove();
+
+    Toastify({
+        text: "Post Successfully Deleted!",
+        className: "info",
+        style: {
+          background: "linear-gradient(to center, #0FA958, #0FA958)",
+        }
+      }).showToast();
+  }
+    // Get the Form modal
+    let formModal = document.getElementById("form-container");
+    let form = document.getElementById("postForm");
+    let postTitle = document.getElementById("post_title")
+    let postText = document.getElementById("post_text")
+    let postImage = document.getElementById("img")
+    let updateButton = document.getElementById("update");
+    // Get the button that opens the Form modal
+    const openFormModal = (key) =>{
+
+    formModal.style.display = "block";
+    // console.log("record To update",key)
```

Comment:
if you are not using these commented lines, please prefer to remove it

Reply:
done.